src/components/KuadrantTLSCreatePage.tsx

```diff
@@ -247,21 +243,13 @@ const KuadrantTLSCreatePage: React.FC = () => {
                   </HelperText>
                 </FormHelperText>
               </FormGroup>
-              <NamespaceSelect
-                selectedNamespace={selectedNamespace}
-                onChange={setSelectedNamespace}
-                formDisabled={formDisabled}
-              >
-              </NamespaceSelect>
               <FormGroup
-                label={t("Gateway API Target Reference")}
                 fieldId="gateway-select"
                 isRequired
               >
                 <GatewaySelect selectedGateway={selectedGateway} onChange={setSelectedGateway} />
               </FormGroup>
               <FormGroup
-                label={t("CertIssuer Issuer Reference")}
```

Comment:
cause of some duplicate labels in the form (the inline elements had same/similar labels)